compiler/control/OMROptions.cpp

```diff
@@ -1616,7 +1616,7 @@ int32_t       OMR::Options::_interpreterSamplingDivisorInStartupMode = -1; // un
 int32_t       OMR::Options::_numJitEntries = 0;
 int32_t       OMR::Options::_numVmEntries = 0;
 int32_t       OMR::Options::_numVecRegsToLock=0;
-int32_t       OMR::Options::_hotFieldThreshold = 100;
+int32_t       OMR::Options::_hotFieldThreshold = 200;
```

Comment:
How was this number derived? and what does it represent?

Reply:
It was derived through performance testing, the value 100 was very low and resulted in a large amount of classes having "hot fields" that weren't actually hot fields. The value 200 isn't optimal as this has not been investigated thoroughly but it resulted in better performance in each benchmark I had tested it with so I bumped up the value to 200.